community/chromium/depends

```diff
@@ -0,0 +1,33 @@
+alsa-lib
```

Comment:
`libXtst` should be added as a dependency.

It is used in `third_party/webrtc/modules/desktop_capture/linux/shared_x_display.cc`